funnel/templates/project.html.jinja2

```diff
@@ -69,13 +69,41 @@
     </div>
   </div>
 
-  <div class="grid about-ticket-wrapper">
-    <div class="{% if project.boxoffice_data and project.boxoffice_data.item_collection_id -%}grid__col-sm-6 grid__col-md-7 {% elif project.allow_rsvp -%}grid__col-sm-6{% else %}grid__col-sm-12{% endif %}" id="about">
-      <h2 class="mui--text-display1 mui--text-left project-section__headline">About</h2>
-      <div class="project-section__description">{{ project.description|safe }}</div>
+  <div class="about">
+    {% if project.allow_rsvp or project.boxoffice_data and project.boxoffice_data.item_collection_id -%}
+      <div class="about__participate">
+        <div class="mui--hidden-md mui--hidden-lg mui--hidden-xl about__participate__back-nav">
+          <div class="mobile-nav mui--z1">
+            <a href="{{ project.url_for() }}" aria-label="{% trans %}Back to the project{% endtrans %}" class="mui--text-dark mui--text-headline" id="close-ticket-widget"><i class="material-icons mui--align-middle mobile-nav__prev" aria-hidden="true">arrow_back</i></a> <span class="mui--text-dark mui--text-headline">Tickets</span>
+          </div>
+        </div>
+        <div id="tickets" {% if project.allow_rsvp -%}class="rsvp-wrapper"{%- endif %}>
+          {% if project.boxoffice_data and project.boxoffice_data.item_collection_id -%}
+            <h2 class="mui--text-display1 mui--text-left project-section__headline">Tickets</h2>
+            <div id="boxoffice-widget"><p class="mui--text-body1">Loading...</p></div>
+          {% elif project.allow_rsvp -%}
+            <h2 class="mui--text-display1 mui--text-left project-section__headline">{% trans %}Attending this event?{% endtrans %}</h2>
+            {%- if current_auth.is_authenticated %}
+              {{ rsvpform(project, rsvp_form) }}
+            {%- else %}
+              <p>
+                <a href="{{ url_for('login') }}" class='mui-btn mui-btn--raised mui-btn--primary' data-action="Login to rsvp">{% trans %}Login to RSVP{% endtrans %}</a>
```

Comment:
I think we should move away from "RSVP" since most people won't know the expansion. _Répondez s'il vous plaît_ (in French). Since it literally means "please respond", maybe we should switch to English. This one can read "Login to respond" (with `{% trans %}`).

Reply:
Wikipedia even says: "The initialism 'RSVP' is no longer used much in France, where it is considered formal and old-fashioned." (hah!)